### ISSUE TITLE
[AVM Module Issue]: output from second plan are not 'clean'

### ISSUE THREAD

**Issue by kewalaka** [Azure/terraform-azurerm-avm-res-app-managedenvironment#37]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Module specific issue

- [X] I confirm that this issue is about this AVM module and not about the resource itself, or how to use Terraform.

### Issue Type?

Bug

### (Optional) Module Version

0.1.2

### Description

when running terraform plan after a successful apply, some of the outputs need to be "changed"

```
     ~ output                    = jsonencode(
            {
              - properties = {
                  - customDomainConfiguration   = {
                      - certificatePassword        = null
                      - certificateValue           = null
                      - customDomainVerificationId = "REDACTED"
                      - dnsSuffix                  = null
                      - expirationDate             = null
                      - subjectName                = null
                      - thumbprint                 = null
                    }
                  - defaultDomain               = "REDACTED.australiaeast.azurecontainerapps.io"
                  - infrastructureResourceGroup = "REDACTED"
                  - peerAuthentication          = {
                      - mtls = {
                          - enabled = false
                        }
                    }
                  - staticIp                    = "10.1.1.150"
                  - vnetConfiguration           = {
                      - dockerBridgeCidr       = null
                      - infrastructureSubnetId = "/subscriptions/REDACTED/resourceGroups/REDACTED/providers/Microsoft.Network/virtualNetworks/REDACTED/subnets/REDACTED"
                      - internal               = true
                      - platformReservedCidr   = null
                      - platformReservedDnsIP  = null
                    }
                  - workloadProfiles            = [
                      - {
                          - name                = "Consumption"
                          - workloadProfileType = "Consumption"
                        },
                    ]
                }
            }
 ```

A plan after apply should report "no changes required"

**Comment by kewalaka** [comment]
Ref https://github.com/Azure/terraform-provider-azapi/issues/444#issuecomment-2033633513

A known issue the Azapi team are planning to address

**Comment by kewalaka** [comment]
fixed by #48
